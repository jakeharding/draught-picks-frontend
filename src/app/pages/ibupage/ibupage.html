<!--
  Generated template for the IbupagePage page.

  See http://ionicframework.com/docs/components/#navigation for more info on
  Ionic pages and navigation.
-->

<!--<draught-header></draught-header>-->
<ion-content padding>
  <ion-row>
    <ion-col class="below-header" text-center col-12 col-sm-10 offset-sm-1 col-md-8 offset-md-2 col-lg-6 offset-lg-3 col-xl-4 offset-xl-4>
      <h1>International Bitterness Units (IBU)</h1>
      <p margin>International Bitterness Units (IBU), is defined as the standard measurement of how bitter your beer is based on the hops contained
        in the drink. According to berghoffbeer.com, IBU "gauges the isomerized alpha acids from hops in a beer in parts per million."
        It is measured on a scale between 0-100 and found on most alcoholic beverage labels. The higher the count
        the more bitter tasting the drink will be.</p>

      <p> <b><u>Example IBU for beers:</u></b></p>
      <ion-row>
      <ion-col offset-2>
        <ion-row>
          <ion-col text-center>Budweiser:</ion-col>
          <ion-col text-left>7 IBU</ion-col>
        </ion-row>

        <ion-row>
          <ion-col text-center>American Brown Ale:</ion-col>
          <ion-col text-left>20 IBU</ion-col>
        </ion-row>

        <ion-row>
          <ion-col text-center>Avery's IPA:</ion-col>
          <ion-col text-left>70 IBU</ion-col>
        </ion-row>

        <ion-row>
          <ion-col text-center>Maharaja:</ion-col>
          <ion-col text-left>100 IBU</ion-col>
        </ion-row>
      </ion-col>
      </ion-row>

      <p>This information was summarized by the following links:</p>
      <li><a href="http://berghoffbeer.com/blog/what-do-abv-and-ibu-mean/" target="_blank">http://berghoffbeer.com/blog/what-do-abv-and-ibu-mean/</a></li>
      <br>
      <li margin-bottom><a href="http://www.5280.com/2013/10/beer-lesson-ibu/" target="_blank">http://www.5280.com/2013/10/beer-lesson-ibu/</a></li>

      <br>
      <a class="backToPreferences" routerLink="/tabs/preferences">Back to your Preferences</a>
    </ion-col>
  </ion-row>
</ion-content>
