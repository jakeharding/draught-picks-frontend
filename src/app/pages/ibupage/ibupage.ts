import { Component } from '@angular/core';
import { BasePage } from '../BasePage';

/**
 * Generated class for the IbupagePage page.
 *
 * See https://ionicframework.com/docs/components/#navigation for more info on
 * Ionic pages and navigation.
 */

@Component({
  selector: 'page-ibupage',
  templateUrl: 'ibupage.html',
})
export class IbupagePage extends BasePage {

  /**
   * IbupagePage constructor
   * Empty but required constructor for the IbupagePage class
   */
  constructor() {
    super('ibu-info');
  }
}
