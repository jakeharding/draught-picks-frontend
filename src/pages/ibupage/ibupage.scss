page-ibupage {
  li{
    list-style-type: none;
  }
  #backToPreferences {
    font-size: medium;
  }
}
