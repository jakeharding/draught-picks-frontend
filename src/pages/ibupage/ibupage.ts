import { Component } from '@angular/core';
import { IonicPage, NavController, NavParams } from 'ionic-angular';
import { BasePage } from '../BasePage';

/**
 * Generated class for the IbupagePage page.
 *
 * See https://ionicframework.com/docs/components/#navigation for more info on
 * Ionic pages and navigation.
 */

@IonicPage()
@Component({
  selector: 'page-ibupage',
  templateUrl: 'ibupage.html',
})
export class IbupagePage extends BasePage {

  /**
   * IbupagePage constructor
   * Empty but required constructor for the IbupagePage class
   * */
  constructor(public navCtrl: NavController, public navParams: NavParams) {
    super('ibu-info');
  }

  ionViewDidLoad() {
    console.log('ionViewDidLoad IbupagePage');
  }

}
