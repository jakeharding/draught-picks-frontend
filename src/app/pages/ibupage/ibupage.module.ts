import { NgModule } from '@angular/core';
import { IonicModule } from '@ionic/angular';
import { IbupagePage } from './ibupage';
import { ComponentsModule } from '../../components/components.module';
import { RouterModule } from '@angular/router';

@NgModule({
  declarations: [
    IbupagePage,
  ],
  imports: [
    IonicModule,
    RouterModule.forChild([{path: '', component: IbupagePage}]),
    ComponentsModule,
  ],
})
export class IbupagePageModule {}
