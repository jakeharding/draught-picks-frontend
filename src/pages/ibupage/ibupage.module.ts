import { NgModule } from '@angular/core';
import { IonicPageModule } from 'ionic-angular';
import { IbupagePage } from './ibupage';
import { ComponentsModule } from "../../components/components.module";

@NgModule({
  declarations: [
    IbupagePage,
  ],
  imports: [
    IonicPageModule.forChild(IbupagePage),
    ComponentsModule,
  ],
})
export class IbupagePageModule {}
