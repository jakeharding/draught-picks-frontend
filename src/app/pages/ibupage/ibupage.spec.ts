import { async, ComponentFixture, TestBed } from '@angular/core/testing';
import { IbupagePage } from './ibupage';
import { CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';

/**
 * ibupage.spec.ts
 *
 * Created by jake
 * Created on 11/18/18
 *
 * Test IbupagePage.
 */

describe('IbupagePage', () => {
  let fixture: ComponentFixture<IbupagePage>;
  let instance: IbupagePage;

  beforeEach(async(() => {
    TestBed.configureTestingModule({
      schemas: [CUSTOM_ELEMENTS_SCHEMA],
      declarations: [IbupagePage],
      providers: []
    }).compileComponents();
  }));

  beforeEach(() => {
    fixture = TestBed.createComponent(IbupagePage);
    instance = fixture.debugElement.componentInstance;
  });

  it('should initialize the IbupagePage', () => {
    expect(instance).toBeTruthy();
    expect(fixture).toMatchSnapshot();
  });
});
