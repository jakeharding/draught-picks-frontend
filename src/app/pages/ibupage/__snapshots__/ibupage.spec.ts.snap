// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`IbupagePage should initialize the IbupagePage 1`] = `
<page-ibupage>
  <ion-header>
    <draught-header />
  </ion-header><ion-content
    padding=""
  >
    <ion-row>
      <ion-col
        col-12=""
        col-lg-6=""
        col-md-8=""
        col-sm-10=""
        col-xl-4=""
        offset-lg-3=""
        offset-md-2=""
        offset-sm-1=""
        offset-xl-4=""
        text-center=""
      >
        <h1>
          International Bitterness Units (IBU)
        </h1>
        <p
          margin=""
        >
          International Bitterness Units (IBU), is defined as the standard measurement of how bitter your beer is based on the hops contained in the drink. According to berghoffbeer.com, IBU "gauges the isomerized alpha acids from hops in a beer in parts per million." It is measured on a scale between 0-100 and found on most alcoholic beverage labels. The higher the count the more bitter tasting the drink will be.
        </p>
        <p>
          <b>
            <u>
              Example IBU for beers:
            </u>
          </b>
        </p>
        <ion-row>
          <ion-col
            offset-2=""
          >
            <ion-row>
              <ion-col
                text-center=""
              >
                Budweiser:
              </ion-col>
              <ion-col
                text-left=""
              >
                7 IBU
              </ion-col>
            </ion-row>
            <ion-row>
              <ion-col
                text-center=""
              >
                American Brown Ale:
              </ion-col>
              <ion-col
                text-left=""
              >
                20 IBU
              </ion-col>
            </ion-row>
            <ion-row>
              <ion-col
                text-center=""
              >
                Avery's IPA:
              </ion-col>
              <ion-col
                text-left=""
              >
                70 IBU
              </ion-col>
            </ion-row>
            <ion-row>
              <ion-col
                text-center=""
              >
                Maharaja:
              </ion-col>
              <ion-col
                text-left=""
              >
                100 IBU
              </ion-col>
            </ion-row>
          </ion-col>
        </ion-row>
        <p>
          This information was summarized by the following links:
        </p>
        <li>
          <a
            href="http://berghoffbeer.com/blog/what-do-abv-and-ibu-mean/"
            target="_blank"
          >
            http://berghoffbeer.com/blog/what-do-abv-and-ibu-mean/
          </a>
        </li>
        <br />
        <li
          margin-bottom=""
        >
          <a
            href="http://www.5280.com/2013/10/beer-lesson-ibu/"
            target="_blank"
          >
            http://www.5280.com/2013/10/beer-lesson-ibu/
          </a>
        </li>
        <br />
        <a
          class="backToPreferences"
          routerdirection="back"
          routerlink="/tabs/preferences"
        >
          Back to your Preferences
        </a>
      </ion-col>
    </ion-row>
  </ion-content>
</page-ibupage>
`;
